loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -1863,6 +1863,21 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		builder._createComment(container, data, true);
 		if (data.children.length > 0)
 		{
+			var numberOfReplies = data.children.length - 1;
+			if (numberOfReplies > 0)
+			{
+				var replyNode = L.DomUtil.create('div','loleaflet-annotation-content',$(container).find('.loleaflet-annotation-content-wrapper')[0]);
```

Comment:
why not create a separate class `loleaflet-annotation-reply-count`  for to display reply and use it along side the `loleaflet-annotation-content`. This way you can put the additional required styling there in CSS file. Which will help when in future if we want to apply different themes.